### PR TITLE
Update golang to version 1.12

### PR DESCRIPTION
**Pull request by jakobjakobson13**
No tests have been done.

### CODE REVIEW

**Comment by jlelse** [comment]
Any updates on this?
Also: The current version is 1.12.5

**Comment by jakobjakobson13** [comment]
> Any updates on this?
> Also: The current version is 1.12.5

I updated the pull request. Let's give the mantainers some time to react.

File: org.freedesktop.Sdk.Extension.golang.json [hunk 1]
```diff
@@ -18,32 +18,32 @@
...
+          "sha256": "ff09f34935cd189a4912f3f308ec83e4683c309304144eae9cf60ebc552e7cd8"
         },
         {
           "type": "archive",
           "only-arches": [
             "x86_64"
           ],
-          "url": "https://dl.google.com/go/go1.11.5.linux-amd64.tar.gz",
-          "sha256": "ff54aafedff961eb94792487e827515da683d61a5f9482f668008832631e5d25"
+          "url": "https://dl.google.com/go/go1.12.5.windows-amd64.zip",
```
Comment (barthalion) [inline org.freedesktop.Sdk.Extension.golang.json]: Zip?

Comment (jakobjakobson13) [reply org.freedesktop.Sdk.Extension.golang.json]: Sorry, my mistake

**Comment by jakobjakobson13** [comment]
Any news?

**Comment by jlelse** [comment]
Still no news?

**Comment by jakobjakobson13** [comment]
> Still no news?

Let's see.

File: org.freedesktop.Sdk.Extension.golang.appdata.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (nedrichards) [inline org.freedesktop.Sdk.Extension.golang.appdata.yaml]: this file cannot be YAML. It must be XML.

Comment (nedrichards) [reply org.freedesktop.Sdk.Extension.golang.appdata.yaml]: Well, what I mean more precisely is, yes it *can* be YAML per https://www.freedesktop.org/software/appstream/docs/sect-AppStream-YAML.html but I am genuinely unsurprised that support for that hasn't been tested.

Comment (jakobjakobson13) [reply org.freedesktop.Sdk.Extension.golang.appdata.yaml]: Thanks for the comment. Where do I file a bug for that issue? At flatpak-builder or somewhere else?

I'll revert the changes soon as any issue/pull request will take longer.

**Pull request closed without merging**